modules/data/src/main/java/io/fluo/webindex/data/spark/AccumuloRangePartitioner.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2015 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.webindex.data.spark;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import io.fluo.api.data.Bytes;
+import io.fluo.api.data.RowColumn;
+import org.apache.hadoop.io.Text;
+import org.apache.spark.Partitioner;
+
+public class AccumuloRangePartitioner extends Partitioner {
```

Comment:
Maybe, its more general than Fluo.  Its more of an Accumulo+Spark integration thing. There is no other good place to put it I suppose.  Would be nice to be able to easily reuse it.